ethereum/remix-project
# Support Proxy

closes https://github.com/ethereum/remix-project/issues/2414

### Testing
* Build and run support proxy PR
* Create an upgradeable smart contract using Openzeppelin wizard by checking the upgradeability UUPS option. https://docs.openzeppelin.com/contracts/4.x/wizard
* Copy and paste the generated code into remix-ide.
* Compile the smart contract and open the Udapp plugin.
* Select the deploy with proxy option
![](https://user-images.githubusercontent.com/6050001/169872425-34dbdba6-d8db-4dd6-9cfe-86c6221671c3.png)
* And deploy
* The Udapp should have 2 instances to interact with the contract. The first is the original contract implementation and the second is the proxy deployment.
